.github/workflows/test.yml

```diff
@@ -15,11 +15,19 @@ jobs:
       with:
         python-version: 3.8
 
-    - name: Install Tox
-      run: pip install tox
+    - name: Install Dependencies
+      run: pip install .[lint]
 
-    - name: Run Tox
-      run: TOXENV=lint tox -r
+    - name: Run Black
+      run: black --check --force-exclude=vyper/version.py ./vyper ./tests ./setup.py
+
+    - name: Run flake8
+      run: flake8 ./vyper ./tests
+      if: always()
```

Comment:
what does this do?

Reply:
Currently, the execution order is `black` > `flake8` > `isort`. If `black` flags any errors and raises an exit code, the `lint` test will end, meaning `flake8` and `isort` will not run (for example, see [this run](https://github.com/vyperlang/vyper/runs/4769987813?check_suite_focus=true). The `always()` flag is used to ensure that `flake8` runs regardless whether `black` raises an exit code, and similarly that `isort` runs regardless whether `black` or `isort` raises an exit code.

Reply:
ah neat!